src/elm/Pile.elm

```diff
@@ -0,0 +1,190 @@
+module Pile exposing (..)
+
+import Draggable
+import Draggable.Events
+import Html exposing (Html)
+import Html.Attributes
+import Html.Events
+import Math.Vector2 exposing (Vec2)
```

Comment:
I've been fully qualifying everything in situ. It's a little more verbose but easier to read in place.